tools/src/main/java/com/nedap/archie/query/APathToXPathConverter.java

```diff
@@ -113,52 +107,50 @@ public static String convertWithAntlr(String query) {
      */
     private static void writeTree(StringBuilder output, ParseTree tree, boolean inPredicate) {
 
-
-        for(int i = 0; i < tree.getChildCount(); i++) {
+        for (int i = 0; i < tree.getChildCount(); i++) {
```

Comment:
This is inconsistent with the rest of the file currently, so reformat on the full file please :)